README.md

```diff
@@ -1 +1,81 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+> This is my professional portfolio outlining previous projects
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Live Demo
+
+[Live Demo Link](https://felix45.github.io/portfolio/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+  node js
+  npm
+
+```
+### Setup
+Clone the repository and change the directory to hello-microverse
+
+``` 
+  $ git clone git@github.com:Felix45/portfolio.git
+
+  $ cd portfolio
+
+```
+
+### Install
+Install all project dependencies by running the command below
+ 
+``` 
+  $ npm install
+```
+### Usage
+Start the http application server
+``` 
+  $ http-server
+```
+
+### Deployment
+- [Visit Application Homepage](http://localhost:8080)
+
+
+
+## Author
+
+👤 **Felix Ouma**
+
+- GitHub: [@Felix45](https://github.com/Felix45)
+- Twitter: [@Felix_Atonoh](https://twitter.com/Felix_Atonoh)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/felix-ouma-639766b0/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Felix45/portfolio/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+
```

Comment:
- [ ]  **[REQUIRED CHANGES ]**

- Your acknowledgment is not descriptive enough as it doesn't point to who you're acknowledging.
Instead, you should give a hat tip to  [Microverse](https://bit.ly/MicroverseTN) for giving you the project to work on,  and to [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1), for using their design.

Reply:
Hi, @zieeco 

- The reviews above have been well received and acted upon.